lib/grizzly/supervisor.ex

```diff
@@ -179,6 +184,9 @@ defmodule Grizzly.Supervisor do
 
   @impl Supervisor
   def init(init_args) do
+    options = Options.new(init_args)
+    ZwaveFirmware.maybe_run_zwave_firmware_update(options)
+
     Supervisor.init(children(init_args), strategy: :one_for_one)
```

Comment:
It looks like `options` could be passed to `children/1` to avoid the duplicate call to `Options.new/1` there.